### ISSUE TITLE
Specify Account Store During Password Reset

### ISSUE THREAD

**Issue by omgitstom** [stormpath/stormpath-sdk-php#36]

**Comment by dmunguia** [comment]
I started working on this one.

**Comment by dmunguia** [comment]
Looking at the java sdk I found there's only one password reset method.
```
Account resetPassword(String passwordResetToken, String newPassword);
```
And there are two methods to generate the token required to do the password reset. One of which considers the account store.
```
Account sendPasswordResetEmail(String email) throws ResourceException;
Account sendPasswordResetEmail(String email, AccountStore accountStore) throws ResourceException;
```
The php sdk only has one such method. 
```
public function sendPasswordResetEmail($accountUsernameOrEmail, array $options = array())
```
Just wanted to confirm that the objective is to include the account store in the password reset flow at the point of sending the email with the token by adding the following method:
```
public function sendPasswordResetEmail($accountUsernameOrEmail, $accountStore, array $options = array())
```

**Comment by omgitstom** [comment]
Hi @dmunguia, that is correct.  By default Stormpath will iterate over all the account stores on the application to find the account to send the password reset to.  By specifying the account store to send the password reset email, the developer is saying he knows where the account should be and wants to target that particular account store.  This is important for [multi tenancy](http://docs.stormpath.com/guides/multi-tenant/) support

**Comment by omgitstom** [comment]
Fixed in 1.3.0 beta

**Comment by ecrisostomo** [comment]
Actually added on version 1.4.0.beta
